sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -216,6 +218,7 @@ def __init__(
         self.topic = kwargs.get('topic', None)
         self.created_on = kwargs['created_on']
         self.created_by = kwargs['created_by']
+        self.retention_policy = kwargs['retention_policy']
```

Comment:
Is this required? Customers should not have been instantiating this model themselves, but it would still be safer to use a `get` here.
```suggestion
        self.retention_policy = kwargs.get('retention_policy')
```